packages/legacy/core/App/components/tour/CredentialOfferTourSteps.tsx

```diff
@@ -1,28 +1,28 @@
 import * as React from 'react'
-import { useTranslation } from 'react-i18next'
-import { Text } from 'react-native'
 
-import { useTheme } from '../../contexts/theme'
+import { Text } from 'react-native'
 import { RenderProps, TourStep } from '../../contexts/tour/tour-context'
 
 import { TourBox } from './TourBox'
+import useCommonTourHooks from '../../hooks/modal-tour-props'
+import { DispatchAction } from '../../contexts/reducers/store'
 
 export const credentialOfferTourSteps: TourStep[] = [
   {
     Render: (props: RenderProps) => {
       const { currentTour, currentStep, next, stop, previous } = props
-      const { t } = useTranslation()
-      const { ColorPallet, TextTheme } = useTheme()
+      const { t, ColorPallet, TextTheme, closeModalTour } = useCommonTourHooks()
+
       return (
         <TourBox
           title={t('Tour.CredentialOffers')}
           hideLeft
           rightText={t('Tour.Done')}
-          onRight={stop}
+          onRight={() => closeModalTour(stop, DispatchAction.UPDATE_SEEN_CREDENTIAL_OFFER_TOUR)}
```

Comment:
This goes along with the suggestion above:
```suggestion
          onRight={endTour}
```